src/api/hooks/useApiTx.ts

```diff
@@ -0,0 +1,17 @@
+import {useContext} from 'react';
+import {useApi} from 'context/ChainApiContext';
+import {TxContext, StartConfig} from 'context/TxContext';
+
+type TxConfig = Omit<StartConfig, 'api'>;
+
+export function useApiTx() {
+  const {start} = useContext(TxContext);
+  const {api} = useApi();
+
+  return (config: TxConfig) => {
```

Comment:
if we want to pass the api to use as params we could also do something like this I guess?
```suggestion
  return (config: (api: ApiPromise) => TxConfig) => {
```
and then use like
```
start((api) => ({params: api.....}))
```

Reply:
Sounds good, I'll make it optional so we don't have to pass a function for simple cases